centreon/tests/e2e/features/Cloud-notifications/03-notification-delete/index.ts

```diff
@@ -0,0 +1,137 @@
+import { checkHostsAreMonitored, checkServicesAreMonitored } from 'e2e/commons';
+
+import data from '../../../fixtures/notifications/data-for-notification.json';
+
+beforeEach(() => {
+  cy.startWebContainer();
+  enableNotificationFeature();
+  setBrokerNotificationsOutput({
+    name: 'central-cloud-notifications-output',
+    configName: 'central-broker-master'
+  });
+
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/internal.php?object=centreon_topology&action=navigationList'
+  }).as('getNavigationList');
+  cy.loginByTypeOfUser({ jsonName: 'admin' });
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/latest/configuration/notifications?page=1&limit=10*'
+  }).as('getNotifications');
+
+  cy.addHostGroup({
+    name: data.hostGroups.hostGroup1.name
+  });
+
+  cy.addHost({
+    activeCheckEnabled: false,
+    checkCommand: 'check_centreon_cpu',
+    hostGroup: data.hostGroups.hostGroup1.name,
+    name: data.hosts.host1.name,
+    template: 'generic-host'
+  })
+    .addService({
+      activeCheckEnabled: false,
+      host: data.hosts.host1.name,
+      maxCheckAttempts: 1,
+      name: data.services.service1.name,
+      template: 'Ping-LAN'
+    })
+    .applyPollerConfiguration();
+
+  checkHostsAreMonitored([
+    {
+      name: data.hosts.host1.name
+    }
+  ]);
+
+  checkServicesAreMonitored([
+    {
+      name: data.services.service1.name
+    }
+  ]);
+});
+
+afterEach(() => {
+  cy.stopWebContainer();
+});
+
+Given('a user with access to the Notification Rules page', () => {
+  cy.navigateTo({
+    page: 'Notifications',
+    rootItemNumber: 3,
+    subMenu: 'Notifications'
+  });
+});
+
+Given('the user is on the Notification Rules page', () => {
+  cy.url().should('include', '/configuration/notifications');
+});
+
+Given('a Notification Rule is already created', () => {
+  createNotification(notificationBody);
+});
+
+When('the user selects the delete action on a Notification Rule', () => {
+  cy.get('#Deletenotificationrule').click();
+});
+
+When('the user confirms the deletion', () => {
+  cy.get('[data-testid="Confirm"]').click();
+});
+
+Then(
+  'a success message is displayed and the Notification Rule is deleted from the listing',
+  () => {
+    cy.get('.MuiAlert-message').should(
+      'have.text',
+      'Notification deleted successfully'
+    );
+    cy.wait('@getNotifications');
+    cy.contains(notificationBody.name).should('not.exist');
+  }
+);
+
+Then(
+  'the configured users are no longer notified of status changes for the associated resources once the notification refresh delay has been reached',
+  () => {
+    cy.submitResults([
+      {
+        host: data.hosts.host1.name,
+        output: 'submit_status_1',
+        status: 'down'
+      }
+    ]);
+
+    checkHostsAreMonitored([
+      {
+        name: data.hosts.host1.name,
+        status: 'down',
+        statusType: 'hard'
+      }
+    ]);
+
+    cy.wait(5000);
```

Comment:
you must not wait arbitrary time ?
you should use vscode eslint extension because it will notice you

Reply:
@kduret, this is not an arbitrary time, it's the refresh_delay's value, set in the lua script